packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -479,6 +489,25 @@ public void setColor(@Nullable Integer color) {
     markUpdated();
   }
 
+  @ReactProp(name = "gradientColors")
+  public void setGradientColors(@Nullable ReadableArray gradientColors) {
+    if (gradientColors != null) {
+      ArrayList<Integer> colors = new ArrayList<Integer>();
+
+      for (int i = 0; i < gradientColors.size(); i++) {
+        if (!gradientColors.isNull(i) && gradientColors.getType(i) == ReadableType.Number) {
+          int color = gradientColors.getInt(i);
+          colors.add(color);
+        }
+      }
+      if (colors.size() >= 2) {
+        mGradientColors = colors.stream()
```

Comment:
I would be wary of using `.stream()` I am not entirely sure if this Java API is backported to all the old Android versions we support.

I'd suggest sticking to a simple for loop.